frontend/packages/console-app/package.json

```diff
@@ -50,7 +50,6 @@
       "nodeStatus": "src/components/nodes/status",
       "nodeActions": "src/components/nodes/menu-actions.tsx",
       "oauthConfigDetailsPage": "src/components/oauth-config/OAuthConfigDetailsPage.tsx",
-      "networkPolicyListPage": "src/components/network-policies/multi-network-policy/NetworkPolicyListPage.tsx",
```

Comment:
this was added in [`7f87bd3` (#13772)](https://github.com/openshift/console/pull/13772/commits/7f87bd31e82ad3a3b75c1e80ac0955a492b5b9dc) but subsequently removed in [`726c831` (#13772)](https://github.com/openshift/console/pull/13772/commits/726c8314e0325e195722ab3d74c5da8bf4c6d865), but this reference was not removed